src/ldw-barsPlus.js

```diff
@@ -1248,7 +1263,25 @@ export default {
     var dim1 = g.data.map(function (d) { return d.dim1; });
     if (g.orientation == "H") dim1.reverse();
     g.dScale.domain(dim1);
-    g.mScale.domain([0, d3.max(g.data, function (d) { return (g.normalized ? 1 : d.offset) * g.gridHeight; })]);
+    var max = d3.max(g.data, function (d) {
```

Comment:
isn't this block of code identical to the one starting at 447? maybe you could define it once and just pass it in  in both those cases?